ert_shared/storage/app.py

```diff
@@ -0,0 +1,78 @@
+import os
+import sys
+from fastapi import FastAPI, Request
+from fastapi.exceptions import HTTPException
+from fastapi.exception_handlers import http_exception_handler
+from fastapi.responses import JSONResponse
+from fastapi.security import HTTPBasic
+
+from starlette import status
```

Comment:
starlette is not defined in project requirements

Reply:
Changed to `from fastapi import status`, which exposes Starlette's `status`.